css-fonts-5/Overview.bs

```diff
@@ -93,13 +93,228 @@ Issue(4044):
 <h3 id="font-size-adjust-prop">
 Relative sizing: the 'font-size-adjust' property</h3>
 
-The 'font-size-adjust' property is applied after the 'size-adjust' descriptor.
+	<pre class="propdef">
+	Name: font-size-adjust
+	Value: none | [ ex | cap | ch | ic ]? [ from-font | <<number>> ]
+	Initial: none
+	Applies to: all elements and text
+	Inherited: yes
+	Percentages: N/A
+	Computed value: the keyword ''font-size-adjust/none'', or a map of the specified keywords and their associated <<number>>
+	Animation type: by computed value type
```

Comment:
Is it well defined how one of those animates? Having the keyword animate discretely while the number interpolate doesn't seem it would give particularly interesting results (except in the subset of cases where you're keeping the keyword constant). Or is it all discrete because one component of it is discrete?

Setting things up so that the resulting adjusted font-size is the thing that interpolates smoothly when you go from, say, `ex 0.5` to `cap 0.8` feels like it ought to be doable, but possibly overly complicated given the use cases for animating this, which I suspect are rather limited and gimmicky.

Reply:
Probably should append "if the keyword remains constant, otherwise discrete"

Reply:
Done.